gateway/response/response.go

```diff
@@ -95,3 +96,19 @@ func GetStatusCode(key string) int {
 
 	return 200
 }
+
+//Always returns a valid response json
+func GetResponse(key string) string {
+	if status, ok := statusMap[key]; ok {
+		response, _ := json.Marshal(status)
+		return string(response)
+	}
+	return "{}"
+}
+
+func MakeResponse(msg string) string {
+	resp, _ := json.Marshal(ResponseStatus{Message: msg})
```

Comment:
handle error and remove unnecessary comments

Reply:
done